docs/src/main/asciidoc/container-image.adoc

```diff
@@ -56,7 +56,34 @@ quarkus.jib.jvm-arguments=-agentlib:jdwp=transport=dt_socket\\,server=y\\,suspen
 
 Other base images might provide launch scripts that enable debugging when an environment variable is set, in which case you would set than environment variable when launching the container.
 
-Finally, the `quarkus.jib.jvm-entrypoint` configuration property can be used to completely override the container entry point and can thus be used to either hard code the JVM debug configuration or point to a script that handles the details.
+==== Custom Entrypoint
+
+The `quarkus.jib.jvm-entrypoint` configuration property can be used to completely override the container entry point and can thus be used to either hard code the JVM debug configuration or point to a script that handles the details.
+
+For example, if the base images `ubi8/openjdk-11-runtime` or `ubi8/openjdk-17-runtime` are used to build the container, the entry point can be hard-coded on the application properties file.
+
+.Example application.properties
+[source,properties]
+----
+quarkus.jib.jvm-entrypoint=java,-Dcustom.param=custom_value,-jar,quarkus-run.jar
+----
+
+Or a custom start-up script can be created and referenced on the properties file. This approach works better if there's a need to set application params using environment variables:
+
+.Example application.properties
+[source,properties]
+----
+quarkus.jib.jvm-entrypoint=/bin/sh,run-java.sh
+----
+
+.Example src/main/jib/home/jboss/run-java.sh
```

Comment:
I think a few words on why `/home/jboss` is used here would be complete the picture

Reply:
Thanks @geoand 
Done!